app/admin.go

```diff
@@ -62,20 +58,51 @@ func (a *App) GetLogsSkipSend(page, perPage int) ([]string, *model.AppError) {
 
 		defer file.Close()
 
-		offsetCount := 0
-		limitCount := 0
-		scanner := bufio.NewScanner(file)
-		for scanner.Scan() {
-			if limitCount >= perPage {
-				break
+		stat, err := file.Stat()
+		if err != nil {
+			return nil, model.NewAppError("getLogs", "api.admin.file_read_error", nil, err.Error(), http.StatusInternalServerError)
+		}
+
+		var newLine = []byte{'\n'}
+		var lineCount int
+		var searchPos = int64(-1)
+		var lineEndPos = stat.Size()
+		file.Seek(lineEndPos, os.SEEK_SET)
```

Comment:
Could you just use `file.Seek(0, os.SEEK_END)` and get rid of the `file.Stat`?

Reply:
That's good, thank you! I'll fix that.